cmd/libsnap-confine-private/cleanup-funcs.c

```diff
@@ -24,33 +24,38 @@ void sc_cleanup_string(char **ptr)
 {
 	if (ptr != NULL) {
```

Comment:
`if (ptr != NULL && *ptr != NULL)`

Reply:
Sleeping over this I think strictly this is not needed because free(NULL) is fine. But having it is fine as well :)